ark/docs/content/docs/expressions/index.mdx

```diff
@@ -197,7 +197,7 @@ Add a type-only symbol to an existing type so that the only values that satisfy
 
 ```ts
 // @noErrors
-const Even = type("(number % 2)#even")
+const even = type("(number % 2)#even")
```

Comment:
Thanks for raising this!

Current convention is actually for these types to be capitalised (as on L238 etc.).

Would you mind adjusting this to make the two examples capitalised?